cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -491,19 +496,38 @@ construct_runtime!(
 		Utility: pallet_utility::{Pallet, Call, Event} = 40,
 		Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 36,
 
-		// Rococo and Wococo BridgeHubs are sharing the runtime, so this runtime has two sets of
-		// bridge pallets. Both are deployed at both runtimes, but only one set is actually used
-		// at particular runtime.
-
-		// With-Wococo bridge modules that are active (used) at Rococo BridgeHub runtime.
+		// Rococo, Wococo and Westend BridgeHubs are sharing the runtime, so this runtime has several sets of
+		// bridge pallets.
+		//
+		// BridgeHubRococo uses:
+		//  - BridgeWococoGrandpa
+		//  - BridgeWestendGrandpa
+		//  - BridgeWococoParachains
+		//  - BridgeWestendParachains
+		//  - BridgeWococoMessages
+		//  - BridgeWestendMessages
+		//  - BridgeRelayers
+		//
+		// BridgeHubWococo uses:
+		//  - BridgeRococoGrandpa
+		//  - BridgeRococoParachains
+		//  - BridgeRococoMessages
+		//  - BridgeRelayers
+
+		// GRANDPA bridge modules.
 		BridgeWococoGrandpa: pallet_bridge_grandpa::<Instance1>::{Pallet, Call, Storage, Event<T>, Config<T>} = 41,
-		BridgeWococoParachain: pallet_bridge_parachains::<Instance1>::{Pallet, Call, Storage, Event<T>} = 42,
-		BridgeRococoToWococoMessages: pallet_bridge_messages::<Instance1>::{Pallet, Call, Storage, Event<T>, Config<T>} = 46,
-
-		// With-Rococo bridge modules that are active (used) at Wococo BridgeHub runtime.
 		BridgeRococoGrandpa: pallet_bridge_grandpa::<Instance2>::{Pallet, Call, Storage, Event<T>, Config<T>} = 43,
-		BridgeRococoParachain: pallet_bridge_parachains::<Instance2>::{Pallet, Call, Storage, Event<T>} = 44,
-		BridgeWococoToRococoMessages: pallet_bridge_messages::<Instance2>::{Pallet, Call, Storage, Event<T>, Config<T>} = 45,
+		BridgeWestendGrandpa: pallet_bridge_grandpa::<Instance3>::{Pallet, Call, Storage, Event<T>, Config<T>} = 48,
+
+		// Parachain bridge modules.
+		BridgeWococoParachains: pallet_bridge_parachains::<Instance1>::{Pallet, Call, Storage, Event<T>} = 42,
+		BridgeRococoParachains: pallet_bridge_parachains::<Instance2>::{Pallet, Call, Storage, Event<T>} = 44,
```

Comment:
I know, I raised this in a parallel bridges-specific PR, but I didn't wanted a rename - just wanted to say that technically, we support bridges with multiple **parachains**, not a single parachain :) I'm fine with the change, but it'll break existing Rococo <> Wococo bridge once it is deployed (pallet name has changed => all storage entries will fade away). Which is also fine, because IIUC we are going to abandon it soon after deploying Rococo <> Westend bridge.

Reply:
yes, this new code will not even be deployed to Rococo<>Wococo, we'll directly deploy Rococo<>Westend and shutdown Wococo network altogether 👍

Reply:
I know :), we could redeploy also Wococo, but we will remove it, so no problem with renaming here.